koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -266,11 +268,12 @@ def bigquery_export_to_partition_mapping(self):
         except GoogleCloudError as err:
             err_msg = (
                 "Could not query table for partition date information."
-                f"\n  Provider: {self._provider_uuid}"
-                f"\n  Customer: {self.customer_name}"
-                f"\n  Response: {err.message}"
+                f"  Provider: {self._provider_uuid}"
+                f"  Customer: {self.customer_name}"
+                f"  Response: {err.message}"
             )
-            LOG.warning(err_msg)
+            extra_context = {"customer": self.customer_name, "response": err.message}
```

Comment:
If we are putting the customer and response into extra_context, it feels redundant to also stick them into the err_msg. Can we do one or the other?